examples/backends/vllm/launch/disagg_flexkv.sh

```
@@ -30,7 +30,7 @@ CUDA_VISIBLE_DEVICES=1 \
   --disaggregation-mode prefill \
   --disable-hybrid-kv-cache-manager \
   --kv-transfer-config '{"kv_connector":"PdConnector","kv_role":"kv_both","kv_connector_extra_config":{"connectors":[{"kv_connector":"FlexKVConnectorV1","kv_role":"kv_both"},{"kv_connector":"NixlConnector","kv_role":"kv_both"}]},"kv_connector_module_path":"kvbm.vllm_integration.connector"}' \
-  --kv-events-config '{"publisher":"zmq","topic":"kv-events","endpoint":"tcp://*:20081","enable_kv_cache_events":true}' &
+  --kv-events-config "{\"publisher\":\"zmq\",\"topic\":\"kv-events\",\"endpoint\":\"tcp://*:${DYN_VLLM_KV_EVENT_PORT:-20081}\",\"enable_kv_cache_events\":true}" &
```

Shellcheck (0.11.0)

[warning] This expansion will not see the mentioned assignment.

(SC2098)